screens/visualization/__init__.py

```diff
@@ -1,3 +1,4 @@
 # Licensed under the GPLv3 - see LICENSE
 
 from .thetatheta import ThetaTheta   # noqa
+from .sketch import sketch   # noqa
```

Comment:
Hmm, probably better not to override the name. Maybe best to change the name of the function.

Also, let's import it here in a PR in which it actually gets used.

Reply:
I started working on the documentation (where `make_sketch` gets used) in a different branch. Did you want that in this PR, or is there some other place where it would be used?